tasks/gen/action.cr

```diff
@@ -0,0 +1,53 @@
+require "colorize"
+require "ecr"
+require "file_utils"
+
+class Lucky::ActionGenerator
+  getter :name
+
+  ECR.def_to_s "#{__DIR__}/action.ecr"
```

Comment:
Instead of using ECR, could you switch this to using teeplate? It's included with LuckyCLI and it's what is used elsewhere to generate files. 

* https://github.com/mosop/teeplate
* Example in LuckyCLI: https://github.com/luckyframework/lucky_cli/blob/master/src/lucky_cli/src_template.cr

Reply:
Excellent.  I saw that Amber is using teeplate and it looks pretty cool.

Reply:
Yeah it's super nice. I didn't want to add it as a dependency on LuckyMigrator, but since Lucky CLI generates a lot of files, it made sense to include it :)